src/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Button from "../Button/Button";
+import Logo from "../../assets/images/logo/logo-white.png";
+import "./PageHeader.scss";
+
+const PageHeader = ({ headerType, heading, text, optionsArr }) => {
+  const optionsJSX = optionsArr.map((option, index) => {
+    return (
+      <option className="option" value={option} key={index}>
+        {option}
+      </option>
+    );
+  });
+
+  const HeadingOnlyJSX = () => (
+    <div className="heading-only">
+      <img src={Logo} alt="logo" className="heading-only__logo" />
+      <h1 className="heading-only__heading">{heading}</h1>
+    </div>
+  );
+
+  const HeadingButtonJSX = () => (
+    <div className="heading-button">
+      <img src={Logo} alt="logo" className="heading-button__logo" />
+      <h1 className="heading-button__heading">{heading}</h1>
+      <Button buttonText={text} />
+    </div>
+  );
+
+  const HeadingButtonDropdownJSX = () => (
```

Comment:
Change this function to include above map. Change parenthesises to {    }  so that is a function and then set the current html into a return ( ... )